<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotusSingleStep</class>
 <widget class="QDialog" name="MotusSingleStep">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="zongqinglabel">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>90</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>纵倾</string>
   </property>
  </widget>
  <widget class="QLabel" name="hengyaolabel">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>90</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>横摇</string>
   </property>
  </widget>
  <widget class="QLabel" name="hangxianglabel">
   <property name="geometry">
    <rect>
     <x>198</x>
     <y>90</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>航向</string>
   </property>
  </widget>
  <widget class="QLabel" name="hengyilabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>横移</string>
   </property>
  </widget>
  <widget class="QLabel" name="qianchonglabel">
   <property name="geometry">
    <rect>
     <x>343</x>
     <y>90</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>前冲</string>
   </property>
  </widget>
  <widget class="QLabel" name="shengjianglabel">
   <property name="geometry">
    <rect>
     <x>418</x>
     <y>90</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>升降</string>
   </property>
  </widget>
  <widget class="QLabel" name="poslabel">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>110</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>位置</string>
   </property>
  </widget>
  <widget class="QLabel" name="speedlabel">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>140</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>速度</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="zongqingposEdit">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>110</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="hengyaoposEdit">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>110</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="hangxiangposEdit">
   <property name="geometry">
    <rect>
     <x>186</x>
     <y>110</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="hengyiposEdit">
   <property name="geometry">
    <rect>
     <x>258</x>
     <y>110</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="qianchongposEdit">
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>110</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="shengjiangposEdit">
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>110</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="zongqingspeedEdit">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>140</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="hengyaospeedEdit">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>140</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="hangxiangspeedEdit">
   <property name="geometry">
    <rect>
     <x>186</x>
     <y>140</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="hengyispeedEdit">
   <property name="geometry">
    <rect>
     <x>258</x>
     <y>140</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="qianchongspeedEdit">
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>140</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="shengjiangspeedEdit">
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>140</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="carryOutButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>执行</string>
   </property>
  </widget>
  <widget class="QLabel" name="posremarklabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>位置：纵倾、横摇、航向单位为度，横移、前冲、升降单位为毫秒</string>
   </property>
  </widget>
  <widget class="QLabel" name="speedremarklabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>速度：纵倾、横摇、航向单位为度/秒，横移、前冲、升降单位为毫米/秒</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
